packages/nuxt/src/head/module.ts

```diff
@@ -7,7 +7,7 @@ const components = ['NoScript', 'Link', 'Base', 'Title', 'Meta', 'Style', 'Head'
 
 export default defineNuxtModule<NuxtOptions['unhead']>({
   meta: {
-    name: 'meta',
+    name: 'unhead',
```

Comment:
Let's add a configKey property instead, and rename the module in a separate PR...